### PR TITLE
Translations for "Integrate your technology with AMP"

### PR DESCRIPTION
**Pull request by bt-amp**
Translations for "Integrate your technology with AMP" (id,ja,ko,pt_BR,zh_CN)

### CODE REVIEW

**Comment by bpaduch** [comment]
@bt-amp - Please ensure you've signed the CLA.
cc: @coryphwang

**Comment by bt-amp** [comment]
Hi Barb,

I filled it out for "only yourself" as the other option required a lot of
other details that I did not have.
*https://screenshot.googleplex.com/1Pjjybkvt28
<https://screenshot.googleplex.com/1Pjjybkvt28>*

Would that be fine?

Thanks,
BT

**Comment by bpaduch** [comment]
Looks like your CLA is good.

File: content/docs/contribute/integrate-your-tech@zh_CN [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
```
Comment (bpaduch) [inline content/docs/contribute/integrate-your-tech@zh_CN]: This file doesn't have a file extension of .md.

**Pull request merged**